floss/language/identify.py

```diff
@@ -26,8 +26,6 @@ def identify_language(sample: Path, static_strings: Iterable[StaticString]) -> L
     Identify the language of the binary given
     """
     if is_rust_bin(static_strings):
-        logger.warning("Rust binary detected, Rust binaries are not supported yet; results may be inaccurate")
-        logger.warning("Rust: proceeding with analysis may take a long time")
```

Comment:
Should we keep these messages around for the string decoding part? We only handle static strings better now.